src/sscofpmf.adoc

```diff
@@ -183,7 +183,7 @@ M-mode).
 
 Read access to bit _X_ is subject to the same mcounteren (or mcounteren and
 hcounteren) CSRs that mediate access to the hpmcounter CSRs by S-mode (or
-VS-mode). In M and S modes, scountovf bit _X_ is readable when mcounteren bit
+VS-mode). In M mode, scountovf bit X is always readable, in S/HS modes, scountovf bit _X_ is readable when mcounteren bit
```

Comment:
Italicize the X, and start a new sentence after "readable" rather than using a comma.  LGTM otherwise.

Reply:
Fixed manually, as I need to make some more edits to this chapter and I don't want to create merge conflicts.